make_queue/views/reservation/reservation.py

```diff
@@ -298,7 +298,8 @@ def form_valid(self, form):
 
         periods = []
         for machine in Machine.objects.filter(machine_type=form.cleaned_data["machine_type"]):
-            periods += self.get_periods(machine, required_time)
+            if not machine.get_status() in "OM":
```

Comment:
I dont think we should exclude machines under maintenance. Maintainance is short-term and the machine is expected to be operational soon. If a machine is out of order it may be long-term, so it makes sense to exclude them.

Reply:
Yes, good point. I'll include the machines under maintenance again